### ISSUE TITLE
Fail when build on windows with nodejs 4.1

### ISSUE THREAD

**Issue by azastrael** [iron-meteor/iron-cli#209]
Hi !
When I try to install the package this fail with this messages ( sorry if some parts are in french but i can translate if you want it ) : 

```
λ npm i -g iron-meteor                                                                                                                                                                                                                        
                                                                                                                                                                                                                                              
> fibers@1.0.7 install C:\Users\matthieu\AppData\Roaming\npm\node_modules\iron-meteor\node_modules\fibers                                                                                                                                     
> node build.js || nodejs build.js                                                                                                                                                                                                            
                                                                                                                                                                                                                                              
/                                                                                                                                                                                                                                             
C:\Users\matthieu\AppData\Roaming\npm\node_modules\iron-meteor\node_modules\fibers {git}{hg}                                                                                                                                                  
{lamb} if not defined npm_config_node_gyp (node "D:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild --release )  else (node  rebuild --release )                                 
Génération des projets individuellement dans cette solution. Pour activer la génération en parallèle, ajoutez le commutateur "/m".                                                                                                            
C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Microsoft.CppBuild.targets(366,5): warning MSB8003: Could not find WindowsSDKDir variable from the registry.  TargetFrameworkVersion or PlatformToolset may be set to an invalid versi 
on number. [C:\Users\matthieu\AppData\Roaming\npm\node_modules\iron-meteor\node_modules\fibers\build\fibers.vcxproj]                                                                                                                          
  fibers.cc                                                                                                                                                                                                                                   
  coroutine.cc                                                                                                                                                                                                                                
c:\users\matthieu\.node-gyp\4.1.0\include\node\v8.h(18): fatal error C1083: Impossible d'ouvrir le fichier include : 'stddef.h' : No such file or directory (compilation du fichier source ..\src\fibers.cc) [C:\Users\matthieu\AppData\Roami 
ng\npm\node_modules\iron-meteor\node_modules\fibers\build\fibers.vcxproj]                                                                                                                                                                     
c:\users\matthieu\.node-gyp\4.1.0\include\node\v8.h(18): fatal error C1083: Impossible d'ouvrir le fichier include : 'stddef.h' : No such file or directory (compilation du fichier source ..\src\coroutine.cc) [C:\Users\matthieu\AppData\Ro 
aming\npm\node_modules\iron-meteor\node_modules\fibers\build\fibers.vcxproj]                                                                                                                                                                  
gyp ERR! build error                                                                                                                                                                                                                          
gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe` failed with exit code: 1                                                                                                                                          
gyp ERR! stack     at ChildProcess.onExit (D:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:270:23)                                                                                                                
gyp ERR! stack     at emitTwo (events.js:87:13)                                                                                                                                                                                               
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)                                                                                                                                                                                     
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)                                                                                                                                                  
gyp ERR! System Windows_NT 6.3.9600                                                                                                                                                                                                           
gyp ERR! command "D:\\Program Files\\nodejs\\node.exe" "D:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild" "--release"                                                                         
gyp ERR! cwd C:\Users\matthieu\AppData\Roaming\npm\node_modules\iron-meteor\node_modules\fibers                                                                                                                                               
gyp ERR! node -v v4.1.0                                                                                                                                                                                                                       
gyp ERR! node-gyp -v v3.0.3                                                                                                                                                                                                                   
gyp ERR! not ok                                                                                                                                                                                                                               
Build failed                                                                                                                                                                                                                                  
                                                                                                                                                                                                                                              
C:\Users\matthieu\AppData\Roaming\npm\node_modules\iron-meteor\node_modules\fibers {git}{hg}                                                                                                                                                  
{lamb} if not defined npm_config_node_gyp (node "D:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild --release )  else (node  rebuild --release )                                 
Génération des projets individuellement dans cette solution. Pour activer la génération en parallèle, ajoutez le commutateur "/m".                                                                                                            
C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Microsoft.CppBuild.targets(366,5): warning MSB8003: Could not find WindowsSDKDir variable from the registry.  TargetFrameworkVersion or PlatformToolset may be set to an invalid versi 
on number. [C:\Users\matthieu\AppData\Roaming\npm\node_modules\iron-meteor\node_modules\fibers\build\fibers.vcxproj]                                                                                                                          
  fibers.cc                                                                                                                                                                                                                                   
  coroutine.cc                                                                                                                                                                                                                                
c:\users\matthieu\.node-gyp\4.1.0\include\node\v8.h(18): fatal error C1083: Impossible d'ouvrir le fichier include : 'stddef.h' : No such file or directory (compilation du fichier source ..\src\fibers.cc) [C:\Users\matthieu\AppData\Roami 
ng\npm\node_modules\iron-meteor\node_modules\fibers\build\fibers.vcxproj]                                                                                                                                                                     
c:\users\matthieu\.node-gyp\4.1.0\include\node\v8.h(18): fatal error C1083: Impossible d'ouvrir le fichier include : 'stddef.h' : No such file or directory (compilation du fichier source ..\src\coroutine.cc) [C:\Users\matthieu\AppData\Ro 
aming\npm\node_modules\iron-meteor\node_modules\fibers\build\fibers.vcxproj]                                                                                                                                                                  
gyp ERR! build error                                                                                                                                                                                                                          
gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe` failed with exit code: 1                                                                                                                                          
gyp ERR! stack     at ChildProcess.onExit (D:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:270:23)                                                                                                                
gyp ERR! stack     at emitTwo (events.js:87:13)                                                                                                                                                                                               
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)                                                                                                                                                                                     
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)                                                                                                                                                  
gyp ERR! System Windows_NT 6.3.9600                                                                                                                                                                                                           
gyp ERR! command "D:\\Program Files\\nodejs\\node.exe" "D:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild" "--release"                                                                         
gyp ERR! cwd C:\Users\matthieu\AppData\Roaming\npm\node_modules\iron-meteor\node_modules\fibers                                                                                                                                               
gyp ERR! node -v v4.1.0                                                                                                                                                                                                                       
gyp ERR! node-gyp -v v3.0.3                                                                                                                                                                                                                   
gyp ERR! not ok                                                                                                                                                                                                                               
Build failed                                                                                                                                                                                                                                  
npm ERR! Windows_NT 6.3.9600                                                                                                                                                                                                                  
npm ERR! argv "D:\\Program Files\\nodejs\\node.exe" "D:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "i" "-g" "iron-meteor"                                                                                                    
npm ERR! node v4.1.0                                                                                                                                                                                                                          
npm ERR! npm  v2.14.3                                                                                                                                                                                                                         
npm ERR! code ELIFECYCLE                                                                                                                                                                                                                      
                                                                                                                                                                                                                                              
npm ERR! fibers@1.0.7 install: `node build.js || nodejs build.js`                                                                                                                                                                             
npm ERR! Exit status 1                                                                                                                                                                                                                        
npm ERR!                                                                                                                                                                                                                                      
npm ERR! Failed at the fibers@1.0.7 install script 'node build.js || nodejs build.js'.                                                                                                                                                        
npm ERR! This is most likely a problem with the fibers package,                                                                                                                                                                               
npm ERR! not with npm itself.                                                                                                                                                                                                                 
npm ERR! Tell the author that this fails on your system:                                                                                                                                                                                      
npm ERR!     node build.js || nodejs build.js                                                                                                                                                                                                 
npm ERR! You can get their info via:                                                                                                                                                                                                          
npm ERR!     npm owner ls fibers                                                                                                                                                                                                              
npm ERR! There is likely additional logging output above.                                                                                                                                                                                     
                                                                                                                                                                                                                                              
npm ERR! Please include the following file with any support request:                                                                                                                                                                          
npm ERR!     C:\Users\matthieu\iron-cli\npm-debug.log                                                                                                                                                                                        
```

Thanks for your help !

**Comment by azastrael** [comment]
Fixed : I switched to nodejs 0.12.7 and it works !

**Comment by NicolasParada** [comment]
There is a lot of problems with node-gyp https://github.com/nodejs/node-gyp/issues Check there.
Make sure you have Visual Studio with the C++ support, and python 2 installed.
I'm currently fine working with node 4.1, windows 10 and iron-meteor :)
